packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -271,7 +310,7 @@ export const RichTextSendBox = (props: RichTextSendBoxProps): JSX.Element => {
       /* @conditional-compile-remove(rich-text-editor-image-upload) */ imageSrcArray?: Array<string>
     ) => {
       /* @conditional-compile-remove(rich-text-editor-image-upload) */
-      cancelInlineImageUpload(imageSrcArray, imageUploadsInProgress, onCancelInlineImageUpload);
+      cancelInlineImageUpload(imageSrcArray, imageUploadsInProgress, undefined, undefined, onCancelInlineImageUpload);
```

Comment:
Can we use named arguemnts here to make it more clear what items are set to undefined.